recipes-core/cve-update/cve-update-nvd2-native.bb

```diff
@@ -37,18 +37,55 @@ CVE_DB_TEMP_FILE ?= "${CVE_CHECK_DB_DIR}/temp_nvdcve_2.db"
 
 CVE_CHECK_DB_FILE ?= "${CVE_CHECK_DB_DIR}/nvdcve_2.db"
 
+CVE_DB_DOWNLOAD_TEMP_FILE = "downlaod_temp_nvdcve_2.db"
+
 python () {
     if not bb.data.inherits_class("cve-check", d):
         raise bb.parse.SkipRecipe("Skip recipe when cve-check class is not loaded.")
 }
 
+def run_download_prepared_db_file(d):
+    base_url = d.getVar("NVDCVE_PREPARED_DATABASE_URL")
+    if len(base_url) == 0:
+        bb.warn("NVDCVE_PREPARED_DATABASE_URL is not set")
+        return False
+
+    url = base_url  + ";downloadfilename=" + d.getVar("CVE_DB_DOWNLOAD_TEMP_FILE")
+
+    try:
+        bb.plain('Fetching prepared NVD database file from ... %s' % url)
+        fetcher = bb.fetch2.Fetch([url], d)
+        fetcher.download()
+    except bb.fetch2.BBFetchException as e:
+        bb.warn("Failed to fetch prepared NVD database file: %s" % e)
+        return False
+
+    return True
+
+def download_prepared_db_file (d):
```

Comment:
This should be "predownload_db_file".

Reply:
@hiraku-wfs all fixed and pushed new commit.